pages/profile-edit/profile-edit.jsx

```diff
@@ -5,20 +5,29 @@ import { Form } from 'react-formbuilder';
 import HintMessage from '../../components/hint-message/hint-message.jsx';
```

Comment:
Most of the code and logic are very similar to what we have for the "add new entry" page. The part that's different is the profile edit form should be prefilled with user's current profile data (which is done via `loadCurrentProfile()`.